.github/workflows/test.yml

```diff
@@ -0,0 +1,11 @@
+name: test
+on: push
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - run: scripts/ci-test.sh 
```

Comment:
notice that there's not much here; we're not tightly coupled to the actual CI system yet (though Actions is nice enough in my experience)